backends/advanced/src/advanced_omi_backend/processors.py

```
@@ -445,12 +458,268 @@ def cleanup_processing_tasks(self, client_id: str):
+                    if client_type == "upload":
+                        # Upload clients: Clean up immediately when processing completes
+                        clients_to_remove.append((client_id, "completed_upload"))
+                        logger.info(f"Marking completed upload client for immediate cleanup: {client_id}")
+
+                        # Also trigger client state cleanup for upload clients
+                        try:
+                            from advanced_omi_backend.main import cleanup_client_state
+                            import asyncio
+
+                            # Schedule client cleanup
+                            asyncio.create_task(self._cleanup_upload_client_state(client_id))
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)